File: examples/basic_test.R

print("About to install the library")

install.packages("academictorrents")



print("About to import the library")

require(academictorrents)




print("About to start a download")

filename = at.get("323a0048d87ca79b68f12a6350a57776b6a3b7fb")

File: examples/basic_test.py


import academictorrents as at
import pickle
import gzip
import sys
import os
import time
import numpy as np

print("Libraries imported, about to start a download")

filename = at.get("55a8925a8d546b9ca47d309ab438b91f7959e77f")

print("About to open the file")

mnist = gzip.open(filename, 'rb')
train_set, validation_set, test_set = pickle.load(mnist)
mnist.close()
